airflow/api_connexion/schemas/dag_run_schema.py

```diff
@@ -104,6 +104,14 @@ def autofill(self, data, **kwargs):
         return data
 
 
+class SetDagRunStateFormSchema(Schema):
+    """Schema for handling the request of setting state of DAG run"""
+
+    run_id = auto_field(data_key='dag_run_id')
+    dag_id = auto_field(dump_only=True)
+    state = DagStateField(dump_only=True)
```

Comment:
```suggestion
class SetDagRunStateFormSchema(Schema):
    """Schema for handling the request of setting state of DAG run"""

    state = DagStateField()
```

We don’t have these fields in the schema now. Also `dump_only` means this field cannot be include in a POST form, which is opposite to what you want. Technically you can add `load_only=True`, but since this entire schema is only used to load data anyway, the flag would not practically change anything.

Reply:
> Also you will still need to satisfy the linter. Again, please consider setting up `pre-commit`.

run the static checks locally, I think it should be good now (for some reason they don't run automatically here in the GitHub actions), thanks!